src/components/Header/HeaderView.js

```diff
@@ -2,13 +2,20 @@ import React from 'react';
 import { Header } from 'bento-components';
 import headerData from '../../bento/globalHeaderData';
 
+const nihLogoImg = {
+  height: '54px',
+  width: '463px',
+  marginLeft: '9px',
+};
+
 const ICDCHeader = () => (
   <>
     <Header
       logo={headerData.globalHeaderLogo}
       easter={headerData.globalHeaderImage}
       alt={headerData.globalHeaderLogoAltText}
       homeLink={headerData.globalHeaderLogoLink}
+      nihLogoImg={nihLogoImg}
```

Comment:
@jonkiky  Can you rename the prop name so that we can easily understand its a styling property